app/app.js

```diff
@@ -10,16 +10,6 @@ import theme from '@elifesciences/elife-theme'
 import Routes from './routes'
 import * as AuthorDetailsSchema from './components/submission/AuthorDetails/AuthorDetailsSchema'
 
-const pixelGridUnit = parseInt(theme.gridUnit, 10)
-theme.space = [
```

Comment:
Why was this here in the first place? I can't see it being used anywhere.

Reply:
It's used but `grid-styled`.